tools/worker-runner/util/nonmonotonic_test.go

```diff
@@ -0,0 +1,22 @@
+package util
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestSleepUntilWallClockTime(t *testing.T) {
+	until := time.Now().Add(time.Millisecond * 10)
+	require.True(t, SleepUntilWallClock(until, context.Background()))
+	require.GreaterOrEqual(t, time.Now().UnixMilli(), until.UnixMilli())
+}
+
+func TestSleepUntilWallClockTimeCancel(t *testing.T) {
+	until := time.Now().Add(time.Second * 10)
+	ctx, _ := context.WithTimeout(context.Background(), 10*time.Millisecond)
```

Comment:
```
Running golangci-lint..
tools/worker-runner/util/nonmonotonic_test.go:19:7: lostcancel: the cancel function returned by context.WithTimeout should be called, not discarded, to avoid a context leak (govet)
	ctx, _ := context.WithTimeout(context.Background(), 10*time.Millisecond)
	     ^
error Command failed with exit code 1.
```

Reply:
I really wish govet didn't run on tests, but ok :)